doc/ref_arch/openstack/chapters/chapter05.md

```diff
@@ -294,13 +294,21 @@ The Libvirt APIs are documented in https://libvirt.org/html/index.html.
 
 ### 5.3.4. Cyborg
 
-|**OpenStack Service** | **API Version** |
-|----------------------|-----------------|
-| Accelerator: Cyborg  | v2              |
+| **OpenStack Service** | **API Version** |
+|-----------------------|-----------------|
+| Accelerator: Cyborg   | v2              |
 
 Acceleration Service API: https://docs.openstack.org/api-ref/accelerator/v2/index.html
 Please note that the initial version of the [Cyborg API v1.0](https://docs.openstack.org/cyborg/stein/admin/api.html) was deprecated in the OpenStack Train release and will be removed in the Ussuri release.
 
+### 5.3.5. Barbican
+
+| **OpenStack Service**           | **API Version** |
+|---------------------------------|-----------------|
+| OpenStack Key Manager: Barbican | v1              |
```

Comment:
@collivier to be consistent can you delete "OpenStack" from "OpenStack Key Manager"? Thanks

Reply:
fixed in https://github.com/cntt-n/CNTT/pull/1357/commits/ac76bcbcb152d1952127452b7475d5ed2826292d